types/react-native/index.d.ts

```diff
@@ -11,7 +11,7 @@
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //
-// USING: these definitions are meant to be used with the TSC compiler target set to ES6
+// USING: these definitions are meant to be used with the TSC compiler target set to at least ES2015.
```

Comment:
There are still some issues with projects switching to pure ES2015-2017, such as the lack of type definitions for `XMLHttpRequest` and `fetch`. However, it can be done if the project uses its own declarations for those. These changes are to help get us on the path so that we can not require the DOM APIs for compiling projects.